opl/investigator/config.py

```diff
@@ -88,6 +92,12 @@ def load_config(conf, fp):
         conf.decisions_es_server = data["decisions"]["es_server"]
         assert not conf.decisions_es_server.endswith("/")
         conf.decisions_es_index = data["decisions"]["es_index"]
+        if "es_server_user" in data["decisions"]:
```

Comment:
Same here. Please rename `es_server_user` to `decisions_es_server_user` and so on.